### ISSUE TITLE
GB issue with the "1" character

### ISSUE THREAD

**Issue by matthill** [openalpr/openalpr#281]
OCR is showing fairly low confidence with OCR on the GB "1" character

![img1](https://cloud.githubusercontent.com/assets/508260/13063846/6839a394-d41a-11e5-812d-d72046c4082a.jpg)

![img2](https://cloud.githubusercontent.com/assets/508260/13063859/855067d8-d41a-11e5-98d3-f8ce9e4ed539.jpg)

**Comment by EccentricDyslexic** [comment]
What camera was this taken with Matt?  Nice clear picture.

Steve

From: Matthew Hill 
Sent: Tuesday, February 16, 2016 12:30 AM
To: openalpr/openalpr 
Subject: [openalpr] GB issue with the "1" character (#281)

OCR is showing fairly low confidence with OCR on the GB "1" character

**Comment by nglinh** [comment]
Is it possible to use regex for postprocessing this out? Frankly it's hard to read even for human

**Comment by matthill** [comment]
I'm not sure which camera it is.  I'm curious as well, it is nice and clear.  A fellow named BenH on the forum posted the issue.

**Comment by matthill** [comment]
Postprocessing is a good option.  Improving the OCR for GB wouldn't hurt either.

**Comment by matthill** [comment]
GB plate information for reference
[registration_plates.pdf](https://github.com/openalpr/openalpr/files/136505/registration_plates.pdf)

**Comment by vladimirr0708982** [comment]
Matt what u think to train OCR better he will get better results ?

**Comment by matthill** [comment]
Yes, definitely.  I think Ben (from the forum) is planning on doing some of that for the characters that don't have as many samples.

**Comment by kenanadakan** [comment]
Hi matthill,
BenH user solved problem on the forum. You can update GB ocr train data?
This forum: [https://groups.google.com/forum/#!msg/openalpr/wI3QLwqK9lI/sae3VzZcBwAJ;context-place=forum/openalpr](url)
